API/src/main/java/io/github/jwdeveloper/tiktok/data/settings/LiveClientSettings.java

```diff
@@ -86,9 +86,23 @@ public class LiveClientSettings {
     private HttpClientSettings httpSettings;
 
     /**
-     * Optional: Sometimes not every messages from chat are send to TikTokLiveJava to fix this issue you can set sessionId
-     * documentation how to obtain sessionId https://github.com/isaackogan/TikTok-Live-Connector#send-chat-messages
+     * Loop time in milliseconds between pings to TikTok
+     * @apiNote Min: 250 (0.25 seconds), Max: 30000 (30 seconds), Default: 5000 (5 seconds)
      */
+    private long pingTaskTime = 5000;
+
+    public void setPingTaskTime(long pingTaskTime) {
```

Comment:
we should validate this inside TikTokClientBuilder#Validate

Reply:
Moved to TikTokClientBuilder#validate